tools/collect_metrics.py

```diff
@@ -44,6 +45,32 @@
 }
 
 
+class ConversionStatus(Enum):
+    DONE = (1,)
+    FALLBACK = (2,)
```

Comment:
We might need another option - NONE.
FALLBACK is valid is there is some conversion (means at least 1 input variation for this op has status DONE), but this particular input is not covered and falls back to torch.